sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/FormRecognizerApiKeyCredential.java

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer.models;
+
+import com.azure.core.util.logging.ClientLogger;
+
+import java.util.Objects;
+
+/**
+ * API key credential that shared across cognitive services, or restrict to single service.
+ *
+ * <p>Be able to rotate an existing API key</p>
+ * {@codesnippet com.azure.ai.formrecognizer.models.FormRecognizerApiKeyCredential}
+ *
+ */
+public final class FormRecognizerApiKeyCredential {
```

Comment:
You can now use `AzureKeyCredential` in azure-core instead of defining custom key credential.

Reply:
Updated to use `AzureKeyCredential`